packages/esm-patient-chart-app/src/visit/visit-action-items/delete-visit-action-item.component.tsx

```diff
@@ -24,10 +29,12 @@ const DeleteVisitActionItem: React.FC<DeleteVisitActionItemProps> = ({ patientUu
     return null;
   }
 
+  const buttonLabel = compact ? t('delete', 'Delete') : t('deleteVisit', 'Delete visit');
```

Comment:
At least things like `delete` and `edit` we should be able to get from core translations. And that's preferable since we don't have to force translators to retranslate the same string.